ddtrace/tracer/remote_config.go

```diff
@@ -75,6 +76,27 @@ func (t *tags) toMap() *map[string]interface{} {
 	return &m
 }
 
+func (t *tracer) dynamicInstrumentationRCUpdate(u remoteconfig.ProductUpdate) map[string]state.ApplyStatus {
+
+	applyStatus := map[string]state.ApplyStatus{}
+
+	for k, v := range u {
+		log.Debug("Received dynamic instrumentation RC configuration for %s\n", k)
+		applyStatus[k] = state.ApplyStatus{State: state.ApplyStateUnknown}
```

Comment:
This is the correct state to set for the config from the tracer's point of view - we don't know here if the BPF instrumentation was applied correctly or not - but I'm not sure what effect it has on the RC backend.

AFAIK, if configs are acknowledged they are not re-sent by the backend, but I'm not 100% sure. We should check with the RC team, I'll add this to the RFC.